_component_design/key-value-pairs.md

```diff
@@ -23,15 +23,15 @@ last-modified: 2017-11-07
 
 ### When to use
 
-Use key-value pairs when a key, an identifier for data, needs either a text or visual display of that key's corresponding value.
+Use key-value pairs when a key (label) needs either text or a visual display of that key’s corresponding value (data).
 
 ### Best practices
 
-- Ensure key values correspond with the label used for the data.
-- Limit key values to one line.
+- Ensure the key (label) corresponds with the data (value).
+- Limit key-value pairs to one line.
```

Comment:
@araiford I'm not sure this make sense. I could think of a situation where a K:V list needs to display a value that exceeds one line. Two examples: Server information, (Key) IPs with (values) IPv4 address && IPv6 address. Similarly on server information there could be a (Key) ssh-key with (value) Long string that exceeds one line but cannon be truncated. Thoughts?

Reply:
Hmmm... looks like we address the second example I provided in the Two Lines section. So I guess the question is, do we allow a Key with multiple values in the same key-values pair as Keys that have a singular value.